src/coredump/coredump.c

```diff
@@ -1652,26 +1656,34 @@ static int forward_coredump_to_container(Context *context) {
         return 0;
 }
 
-static int gather_pid_mount_tree_fd(const Context *context) {
-        _cleanup_close_ int mntns_fd = -EBADF, root_fd = -EBADF;
+static int gather_pid_mount_tree_fd(const Context *context, int *ret_fd) {
+        /* Don't bother preparing environment if we can't pass it to libdwfl. */
+#if !HAVE_DWFL_SET_SYSROOT
+        *ret_fd = -EOPNOTSUPP;
+        log_debug_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "dwfl_set_sysroot() is not supported.");
```

Comment:
use plain log_debug() if you are making up an error here and are not interested in the return value of log_debug_errno()